src/test/java/duke/DeadlineTest.java

```diff
@@ -0,0 +1,33 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class DeadlineTest {
+
+    /**
+     * Test Deadline with valid input
```

Comment:
maybe the first sentence could start with Tests instead of Test (eg. Returns instead of Return)